lion/io/Xml_element.h

```diff
@@ -62,15 +62,31 @@ class Xml_element
     //! Add comment
     void add_comment(const std::string& text) { _e->InsertNewComment(text.c_str()); }
 
+    //! See if the element has children
+    bool has_children() const {return !_e->NoChildren();}
```

Comment:
Nice

Reply:
This does not work for some reason. Had to change it to check if first child was nullptr or not

Reply:
Hum, ok. Weird, but ok. It's on Tinyxml2 side so never mind